app/src/main/res/layout/list_steps.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        android:layout_width="match_parent"
+        android:orientation="horizontal"
+        android:layout_height="wrap_content">
+
+    <TextView
+            android:id="@+id/txt_short_description"
+            android:padding="@dimen/card_view_margin"
+            android:layout_width="0dp"
+            android:layout_weight="0.8"
```

Comment:
@KhinThiriSoe not sure where to put weight value. I think it should be in dimen resource.

move the width value to dimen resource

Reply:
should I put weight value to dimens?

Reply:
@KhinThiriSoe yes if you can